### PR TITLE
Make the pre-push script work on directories with spaces

### PR DESCRIPTION
**Pull request by jyn514**
As a secondary benefit, it's also a lot simpler.

cc @caass - would love to have a review if you have time :)

### CODE REVIEW

File: src/etc/pre-push.sh [hunk 1]
```diff
@@ -10,16 +10,8 @@ set -Eeuo pipefail
...
-  COMMAND="python3 $COMMAND"
-fi
-
-echo "Running pre-push script '$COMMAND'"
+echo "Running pre-push script $ROOT_DIR/x test tidy"
 
 cd "$ROOT_DIR"
-
-$COMMAND
+./x test tidy
```
Comment (jyn514) [inline src/etc/pre-push.sh:17]: This is already in a .sh file, so we know there must be a shell available.

Comment (caass) [reply src/etc/pre-push.sh:17]: can you just run a python script like this? TIL

Comment (jyn514) [reply src/etc/pre-push.sh:17]: yes. see https://github.com/rust-lang/rust/issues/94829 and https://github.com/rust-lang/rust/issues/71818.

**Comment by Mark-Simulacrum** [comment]
r=me

**Comment by jyn514** [comment]
Thanks!

@bors r+ rollup (pre-push script isn't tested in CI)

**Pull request merged**